configure_check_conditions.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>Add, Edit, or Remove a Condition</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>The word add is tricky, since it's more "add built-in condition"</p>

Reply:
<p>Yes but I think its better to keep it simple. and you can add a sentence about that in the section itself under the "Add Condition"</p><p>btw theoretically they can NameOfCheck.add_condition(lambda_function)</p><p>but that's quite advanced...</p>